python/tk_desktop/desktop_engine_site_implementation.py

```diff
@@ -247,7 +247,8 @@ def run(self, splash, version, **kwargs):
         icon = QtGui.QIcon(":tk-desktop/default_systray_icon")
         app.setWindowIcon(icon)
 
-        splash.set_message("Building UI")
+        if splash:
```

Comment:
This wouldn't be a backwards compatible change with desktop startup, we can't change this at this point.